contrib/builder/deb/amd64/generate.sh

```diff
@@ -42,11 +42,19 @@ for version in "${versions[@]}"; do
 	echo >> "$version/Dockerfile"
 
 	if [ "$distro" = "debian" ]; then
-		cat >> "$version/Dockerfile" <<-'EOF'
-			# allow replacing httpredir mirror
-			ARG APT_MIRROR=httpredir.debian.org
-			RUN sed -i s/httpredir.debian.org/$APT_MIRROR/g /etc/apt/sources.list
-		EOF
+		if [ "$suite" = "stretch" ]; then
+			cat >> "$version/Dockerfile" <<-'EOF'
+				# allow replacing deb mirror
+				ARG APT_MIRROR=deb.debian.org
+				RUN sed -i s/deb.debian.org/$APT_MIRROR/g /etc/apt/sources.list
```

Comment:
I have the feeling we do not need to have the sed pattern be flexible for `httpredir` and `deb`. Only `stretch` is using `deb.debian.org` and it doesn't look like they will switch back.

If things are switching back and forth in the future, I can put in another PR to be flexible. Unless you guys have an example where it would save a good amount of grief, I'd rather keep this PR more directed to `stretch`.